lib/assesments/assessment_2/fi_a2_bilal0x01/screens/home/widgets/story_carousel_bilal0x01.dart

```diff
@@ -0,0 +1,35 @@
+part of '../home_screen_bilal0x01.dart';
+
+class _StoryCarouselBilal0x01 extends StatelessWidget {
+  const _StoryCarouselBilal0x01({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final List<String> _carouselImages = const [
+      StaticAssets.girlImg1,
+      StaticAssets.manImg1,
+      StaticAssets.girlImg2,
+      StaticAssets.manImg2
+    ];
+
+    return SizedBox(
+      height: 60,
+      child: ListView.builder(
+        itemExtent: 100,
+        scrollDirection: Axis.horizontal,
+        itemCount: _carouselImages.length,
+        itemBuilder: (context, index) {
+          return CircularImageBuilderBilal0x01(
+            hasBorder: index == 0 ? false : true,
```

Comment:
simply write 
```hasBorder: index != 0```
this is already a boolean why explicitly write that afterwards? it's not necessary,